Update customer-key-manage.md (#8595)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/customer-key-manage.md

> ### Revoke your Customer Keys and the availability key for SharePoint Online, OneDrive for Business, and Teams files

Reviewer: ```suggestion
### Revoke your Customer Keys and the availability key for SharePoint, OneDrive for work or school, and Teams files
```

> Purging of SharePoint Online, OneDrive for Business and Teams files DEP is not supported for Customer Key. These multi-workload DEP is used to encrypt data across multiple workloads across all tenant users. Purging such DEP would result into data from across multiple workloads become inaccessible. If you decide to exit Microsoft 365 services altogether then you could pursue the path of tenant deletion per the documented process. See how to delete a tenant in Azure Active Directory.  

Reviewer: ```suggestion
Purging of SharePoint, OneDrive for work or school, and Teams files DEPs is not supported in Customer Key. These multi-workload DEPs are used to encrypt data across multiple workloads across all tenant users. Purging such a DEP would result in data from across multiple workloads becoming inaccessible. If you decide to exit Microsoft 365 services altogether, you could pursue the path of tenant deletion per the documented process. See how to [delete a tenant in Azure Active Directory](/azure/active-directory/enterprise-users/directory-delete-howto).  
```
